pkg/cluster/cluster.go

```diff
@@ -513,6 +524,7 @@ func (c *Cluster) Update(newSpec *spec.Postgresql) error {
 	}
 
 	c.setStatus(spec.ClusterStatusRunning)
+	c.setProcessName("")
```

Comment:
I'd really use defer here and in other places where you clear up the status (in case the relevant function exits with an error).

Reply:
makes sense. thank you